caching-service/src/main/java/org/zowe/apiml/caching/service/infinispan/config/InfinispanConfig.java

```diff
@@ -60,6 +60,8 @@ public class InfinispanConfig {
     private String address;
     @Value("${jgroups.keyExchange.port:7601}")
     private String keyExchangePort;
+    @Value("${jgroups.tcp.diag.enabled:true}")
```

Comment:
Why does it have a different default than the one in start.sh?

Reply:
I forgot this one from testing